.github/ISSUE_TEMPLATE/data-migration.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Data migration
-about: (for core team) Track the status of a data migration
+about: (for core team) Track the isEmailSent of a data migration
```

Comment:
@junming403, be careful when using IntelliJ to refactor. This is one of my pet peeves for IntelliJ.